metadata.json

```diff
@@ -33,5 +33,48 @@
       "version_requirement": ">= 7.0.0"
     }
   ],
-  "data_provider": null
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7",
+	"8"
```

Comment:
Indentation looks off here.

Reply:
Well spotted! This was caused by using a tab rather than whitespaces. This is fixed now.